SchemaWrapper.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import SchemaHeader from  "./SchemaHeader";
+import SchemaBody from "./SchemaBody";
+
+type SchemaWrapperProps = {
+    info: any;
+}
+
+const SchemaWrapper:React.FC<SchemaWrapperProps> = ({ info }) => {
+    const { title, description, auth_required, auth_scopes, properties } = info;
```

Comment:
Instead of using the `info: any` prop, we could perhaps specify the destructured variable with types.

I think we can do:
```
const SchemaWrapper:React.FC = ({ info }) => {

  interface infoData {
    title: string
    description: string
   ....
  }
  
  const info_data: infoData = info

  ...
}

```